vllm/engine/llm_engine.py

```diff
@@ -478,7 +487,7 @@ def _process_sequence_group_outputs(self, seq_group: SequenceGroup,
             child_seqs.append((parent, parent))
 
         for seq, _ in child_seqs:
-            if seq_group.sampling_params.detokenize:
+            if seq_group.sampling_params.detokenize and self.detokenizer:
```

Comment:
Don't we need to do add changes to the following similar to what you did here as well?
https://github.com/vllm-project/vllm/blob/0ce0539d4750f9ebcd9b19d7085ca3b934b9ec67/vllm/engine/llm_engine.py#L435-L437

Otherwise we will get a error when `self.detokenizer` is `None`